contentcuration/contentcuration/serializers.py

```diff
@@ -557,16 +564,41 @@ def retrieve_metadata(self, node):
                 "has_changed_descendant": node.changed,
             }
 
+    def retrieve_source_channel(self, node):
+        # TODO(davidhu): This incurs an extra DB query -- how can we avoid?
```

Comment:
@micahscopes has some pretty cool work with subqueries that might be able to get around this. ([for reference](https://docs.djangoproject.com/en/2.0/ref/models/expressions/#subquery-expressions)) No need to add this right away, but might be good to add in the future

Reply:
Great! I might do this in a future PR — seems doable enough as a subquery.